projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx

```diff
@@ -101,7 +101,13 @@ function SubscriptionsSettings( props ) {
 			module={ SUBSCRIPTIONS_MODULE_NAME }
 			header={ __( 'Subscriptions', 'jetpack' ) }
 		>
-			<SettingsGroup disableInOfflineMode disableInSiteConnectionMode module={ subscriptions }>
+			<SettingsGroup disableInOfflineMode disableInSiteConnectionMode module={ subscriptions } >
```

Comment:
Let's remove the trailing space.
```suggestion
			<SettingsGroup disableInOfflineMode disableInSiteConnectionMode module={ subscriptions }>
```